builder/rootfs/bin/boot

```diff
@@ -74,6 +88,27 @@ done
 docker build -t deis/slugbuilder /usr/local/src/slugbuilder/
 docker build -t deis/slugrunner /usr/local/src/slugrunner/
 
+function gen_host_keys {
+	# generate the keys, then set them up in etcd if not already set
+	/usr/bin/ssh-keygen -A
```

Comment:
Does it make sense to run `ssh-keygen` before checking if the keys are already there? Seems unnecessary if we're just going to pull from etcd anyhow.